build.gradle

```diff
@@ -79,7 +79,7 @@ ext.deps = [
         lithoTesting       : "com.facebook.litho:litho-testing:$LITHO_VERSION",
         // Debugging and testing
         guava              : 'com.google.guava:guava:20.0',
-        robolectric        : 'org.robolectric:robolectric:3.3',
+        robolectric        : 'org.robolectric:robolectric:4.2',
```

Comment:
We can't do that. :(

This needs to match our internal version.

Reply:
Understood